projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/secrets/service/vault/VaultJobSecretsService.java

```diff
@@ -23,6 +27,9 @@
 @ConditionalOnProperty(value = "featureflag.vault.integration.enabled")
 public class VaultJobSecretsService implements com.vmware.taurus.secrets.service.JobSecretsService {
 
+  @Value("${vdk.vault.size.limit}")
```

Comment:
I forgot to mention the last time but why `vdk.` prefix when all other variables use `datajobs.` prefix. 

Technically `vdk.` is better one since this is the product name. 

`datajobs` is more of legacy prefix since the time the control service was called only Data Jobs Service. 

Still would users wonder about the difference ?

Reply:
fixed to be consistent with old properties